flow/connectors/mysql/schema.go

```diff
@@ -0,0 +1,101 @@
+package connmysql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/PeerDB-io/peerdb/flow/generated/protos"
+)
+
+func (c *MySqlConnector) GetAllTables(ctx context.Context) (*protos.AllTablesResponse, error) {
+	rs, err := c.Execute(ctx, `select concat(table_schema, '.', table_name) from information_schema.tables
+		where table_schema not in ('information_schema', 'performance_schema', 'sys')`)
+	if err != nil {
+		return nil, err
+	}
+
+	tables := make([]string, 0, rs.RowNumber())
+	for idx := range rs.RowNumber() {
+		val, err := rs.GetString(idx, 0)
+		if err != nil {
+			return nil, err
+		}
+		tables = append(tables, val)
+	}
+	return &protos.AllTablesResponse{Tables: tables}, nil
+}
+
+func (c *MySqlConnector) GetSchemas(ctx context.Context) (*protos.PeerSchemasResponse, error) {
```

Comment:
I'm not sure if this should even be a thing, because MySQL doesn't have schemas and we're essentially showing tables across all databases with this
```
In MySQL, physically, a schema is synonymous with a database. You can substitute the keyword SCHEMA instead of DATABASE in MySQL SQL syntax, for example using CREATE SCHEMA instead of CREATE DATABASE.
```

Reply:
Yes, I think it's better here to treat mysql databases like schemas rather than postgres databases

Reply:
then it makes sense to remove database from the config of mysql peers